htop.c

```diff
@@ -53,6 +55,25 @@ static void printHelpFlag() {
    exit(0);
 }
 
+static bool parsePID(char *pid_str, unsigned int *result) {
+   char *endptr;
+   unsigned long tmp_result;
+
+   if (pid_str[0] == '-')
+      return false;
```

Comment:
No it won't. `strtoul` accepts negative numbers but will be cast to unsigned. (Similar to `(unsigned long)stroul(str,endptr,base)` )